swe_bench_util/index/astra_assistants.py

```diff
@@ -30,10 +33,9 @@ def upload_file(file_path) -> str | None:
                 "rb",
             ),
             purpose="assistants",
-            embedding_model="text-embedding-3-large",
```

Comment:
Just curious, does the embedding model default without this?

Reply:
I think default is ada-002. We should get scientific and test out a few out including different providers.